src/services/loop/loop-rest-client.ts

```diff
@@ -0,0 +1,125 @@
+// https://lightning.engineering/loopapi/#loop-rest-api-reference
+import fs from "fs"
+import https from "https"
+
+import axios, { AxiosResponse } from "axios"
+
+import { getLoopConfig } from "@config"
+
+// @todo - get from yaml or .envrc
+const loopMacaroon = convertMacaroonToHexString(
+  fs.readFileSync("./dev/lnd/loop.macaroon"),
+)
+const cert = fs.readFileSync("./dev/lnd/tls.cert")
+const key = fs.readFileSync("./dev/lnd/tls.key")
```

Comment:
this will only work in local environments

Reply:
yep still WIP